src/comparison/tests/unit/test_vcf_pipeline_utils.py

```diff
@@ -0,0 +1,193 @@
+import shutil
+from collections import Counter
+from os.path import basename, exists
+from os.path import join as pjoin
+from pathlib import Path
+from unittest import mock
+from unittest.mock import patch
+
+import pandas as pd
+import pysam
+import pytest
+from simppl.simple_pipeline import SimplePipeline
+from ugbio_comparison.vcf_pipeline_utils import (
+    VcfPipelineUtils,
+    _fix_errors,
+    bed_file_length,
+    close_to_hmer_run,
+    vcf2concordance,
+)
+from ugbio_core.h5_utils import read_hdf
+from ugbio_core.vcfbed import vcftools
+from ugbio_core.vcfbed.interval_file import IntervalFile
+
+
+@pytest.fixture
+def resources_dir():
+    return Path(__file__).parent.parent / "resources"
+
+
+def test_fix_errors(resources_dir):
+    data = read_hdf(pjoin(resources_dir, "h5_file_unitest.h5"), key="concordance")
```

Comment:
```suggestion
    data = read_hdf(resources_dir / "h5_file_unitest.h5", key="concordance")
```

In general, we prefer Path rather than pjoin (when possible)